source/backend/dal/Repositories/ConsultationRepository.cs

```diff
@@ -0,0 +1,92 @@
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+
+namespace Pims.Dal.Repositories
+{
+    /// <summary>
+    /// Provides a repository to interact with consultations within the datasource.
+    /// </summary>
+    public class ConsultationRepository : BaseRepository<PimsLeaseConsultation>, IConsultationRepository
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a ConsultationRepository, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public ConsultationRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<ConsultationRepository> logger)
+            : base(dbContext, user, logger)
+        {
+        }
+
+        #endregion
+
+        #region Methods
+
+        public List<PimsLeaseConsultation> GetConsultationsByLease(long leaseId)
+        {
+            using var scope = Logger.QueryScope();
+
+            return Context.PimsLeaseConsultations
+                .Where(lc => lc.LeaseId == leaseId)
+                .Include(lc => lc.ConsultationTypeCodeNavigation)
```

Comment:
Do we need the ConsultationStatusTypeCodeNavigation as well? or neither?

Reply:
Consultation status is no longer used.